Docs/Whats_New.md

```diff
@@ -26,6 +26,7 @@ Released TBD
 	- `VK_EXT_external_memory_metal`.
 	- `VK_EXT_image_2d_view_of_3d`
 	- `VK_EXT_tooling_info`
+	- `VK_KHR_shader_expect_assume`
```

Comment:
Please move up to just below `VK_KHR_maintenance4`, to keep same sorting order as everywhere else (KHR alpha, then EXT alpha).